### DISCUSSION TITLE
SCEP and Microsoft Intune

### DISCUSSION THREAD

**Discussion by dgitw** [smallstep/certificates#942] [Ideas] [answered]
Given Step CA supports SCEP as a provisioner, I was wondering if anyone had looked at integration with [Microsoft Intune](https://docs.microsoft.com/en-us/mem/intune/protect/certificate-authority-add-scep-overview). This describes the Intune [API to add third party CA](https://docs.microsoft.com/en-us/mem/intune/protect/scep-libraries-apis). Is this a scenario in the pipeline?

**Comment by tashian** [comment, marked as answer]
Hi @dgitw,

Here's [an issue](https://github.com/smallstep/certificates/issues/608) that a community member created to track Intune support in `step-ca`. As we're not focused on MDM at the moment, this integration is not on our short-term roadmap. However, feel free to +1 the issue so we can track the level of community support for this feature.
